src/translations/en/websitetoolboxforum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * websitetoolboxforum plugin for Craft CMS 3.x
+ *
+ * Upgradation of plugin for craft v4
+ *
+ * @link      http://websitetoolbox.com
+ * @copyright Copyright (c) 2022 Websitetoolbox
+ */
+
+/**
+ * @author    Websitetoolbox
+ * @package   Websitetoolboxforum
+ * @since     1.4.0
+ */
+return [
+    'websitetoolboxforum plugin loaded' => 'websitetoolboxforum plugin loaded',
```

Comment:
In this file you say forum, in other files you say community.

?

Reply:
I have made changes in code and replace the forum with community.